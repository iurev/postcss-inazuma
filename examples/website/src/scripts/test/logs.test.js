import * as logs from '../logs';

describe('test logWorld()', () => {
  test('with a number expect to return its square', () => {
    expect(logs.logSquare(3)).toBe(27);
  });

  test('with not a number string expect to return false', () => {
    expect(logs.logSquare('3')).toBe(false);
  });


  test('without a parameter expect to return false', () => {
    expect(logs.logSquare()).toBe(false);
  });
});

describe('test logWorld()', () => {
  test('return greeting sentence', () => {
    expect(logs.logWorld('Sandrina')).toMatchSnapshot();
  });
});
