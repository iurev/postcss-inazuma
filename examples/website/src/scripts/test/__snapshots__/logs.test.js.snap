// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test logWorld() return greeting sentence 1`] = `"Wello Sandrina!"`;
